pandas/core/arrays/datetimes.py

```diff
@@ -158,15 +158,8 @@ def f(self):
             # these return a boolean by-definition
             return result
 
-        if field in self._object_ops:
```

Comment:
Can `_object_ops` be in-lined in `_datetimelike_ops` now since this usage is removed?

Reply:
I think so. Will update.